### PR TITLE
Write some standard installation instructions

### LINKED ISSUE #13: Write about installing in README
We should add a a couple of lines on how to install this, either through `bower` or `npm`.

### PR DESCRIPTION
**Pull request by sindrenm**
Fixes #13. Feel free to add additional stuff that I might have missed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,26 @@ _Resulting layout:_
...
+the appropriate command line tool. If you're using Bower:
+
+```
+$ bower install --save hyper-content-for
+```
+
+If you're using npm, use the `npm` executable instead:
+
+```
+$ bower install --save hyper-content-for
```
Comment (t0ggah) [inline README.md]: If you're using npm, i personally like to use 'npm'  :)

Comment (sindrenm) [inline README.md]: Well poop.

Comment (jgorset) [inline README.md]: Ha ha.

**Comment by jgorset** [comment]
:+1:

**Pull request merged**